### DISCUSSION TITLE
Calculation of popularity for trending posts and news

### DISCUSSION THREAD

**Discussion by playmobilmeister** [mastodon/mastodon#21558] [Q&A]
I was looking the other day for documentation on github regarding how the server decides on candidates for trending posts. on our server (v4) I have some weird behaviour occassionally

Can anyone direct me to som reference about this, please ?

**Comment by just-cameron** [comment]
I'd be interested in this as well -- our instance might like to tinker with the trending alg to show more relevant, long-lasting stuff to our users.

**Comment by Gargron** [comment]
https://github.com/mastodon/mastodon/blob/main/app/models/trends/statuses.rb

**Reply by just-cameron** [reply]
Thank you!
